src/addMessage/addMessage.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import './addMessage.css'
+import { useDispatch, useSelector } from 'react-redux';
+import { addComment } from '@/features/allComments/allComments';
+import { v4 } from 'uuid';
+import moment from 'moment/moment';
+import { allAccountsList } from '@/store/selectors';
+import { changeCurrentUser } from '@/features/currentUser/currentUser';
+const AddMessage = () => {
+    const allAccounts = useSelector(allAccountsList)
+    const [currentTextAreaValue, setCurrentTextAreaValue] = useState('')
+    const [currentUser, setCurrentUser] = useState(allAccounts[0])
+    const dispatch = useDispatch()
+    const addMessageHandler = (event) => {
+        event.preventDefault();
+        const messagetInf = {
+            postId: v4(),
+            userId: currentUser.id,
+            text: currentTextAreaValue,
+            userName: currentUser.accountName,
+            date: moment().format('YYYY-MM-DD HH:mm'),
+            pathToProfilePhoto: currentUser.pathToPhoto,
+            quantityOflikes: 1,
+            answers: []
+        }
+        dispatch(addComment({
+            id: 'no id',
```

Comment:
why do you need to use string `no id` here? if there is no id, then just skip this property
```ts
dispatch(addComment({ newComment: messagetInf }))
```